android/src/av/imageview/AvImageView.java

```diff
@@ -160,13 +163,13 @@ public void setImageAsURL(String uri) {
             this.progressBar.setVisibility(View.VISIBLE);
         }
 
-        if (currentProperties.containsKey("signature") && currentProperties.getString("signature") != "") {
+        if (currentProperties.containsKey("signature") && !currentProperties.getString("signature").isEmpty()) {
```

Comment:
I throws an "isEmpty()" on a null object reference for me. In my case signature was `undefined`. I can change it to `""` in my code to fix it but perhaps you could check for `null` too